README.md

```diff
@@ -1,8 +1,17 @@
-# Celestia Networks Repository
+# Celestia Networks
 
-This repository contains the `genesis.json` for Celestia testnets.
+This repository contains the configuration files for Celestia networks.
 
-If you want to see versions per testnets that are being used, then navigate to this [grafana dashboard](https://celestia.grafana.net/public-dashboards/5d14d96e44f04664bb0c44267e5d645c)
+Please refer to the [Celestia Docs](https://docs.celestia.org) for guides on running your own node. The configuration files `genesis.json` and `addrbook.json` are intended for use by Consensus nodes and should be placed in the node's home directory (by default `~/.celestia-app/config`).
 
-Please refer to the Celestia Docs for guides on getting started running
-your own node [here](https://docs.celestia.org)
+## Networks
+
+| Name     | Type    | Chain ID     | Configs                    |
+|----------|---------|--------------|----------------------------|
+| Mocha    | Testnet | `mocha-4`    | [mocha-4](./mocha-4)       |
+| Arabica  | Testnet | `arabica-10` | [arabica-10](./arabica-10) |
+| Celestia | Mainnet | `celestia`   | [celestia](./celestia)     |
+
+## Software versions
+
+The celestia-app and celestia-node versions in use on each testnet are visible on this [Grafana dashboard](https://celestia.grafana.net/public-dashboards/5d14d96e44f04664bb0c44267e5d645c).
```

Comment:
?
```suggestion
The celestia-app and celestia-node versions in use on each network are visible on this [Grafana dashboard](https://celestia.grafana.net/public-dashboards/5d14d96e44f04664bb0c44267e5d645c).
```

Reply:
Great catch. The current dashboard just has testnet versions but per @Bidon15 's comment, it will soon show mainnet versions.